index.html

```diff
@@ -10,24 +10,35 @@
   <link rel="stylesheet" href="css/main.css">
 </head>
 
-<body class="theme-normal">
+<body class="home theme-normal">
   <nav>
-    <div id="burger" class="burger-container">
-      <span class="burger"></span>
+    <div class="sub-menu">
+      <ul class="sub-menu-social">
+        <li>
+          <a href="#"><div class="face-icon"></div></a>
+        </li>
+        <li>
+          <a href="#"><div class="twit-icon"></div></a>
+        </li>
+        <li><a class="logout" href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="main-menu">
+      <div id="burger" class="burger-container">
+        <span class="burger"></span>
+      </div>
+      <div id="nav-logo" class="event-logo"></div>
```

Comment:
- [ ] **OPTIONAL**
This should be a link `<a>` to the homepage as well as the one on the about page.